tests/rkt_tests.go

```diff
@@ -219,6 +219,19 @@ func createTempDirOrPanic(dirName string) string {
 	return tmpDir
 }
 
+// The createFileOrPanic creates an empty file within the given directory
+// with a specified name. Panics if file creation fails for any reason.
+func createFileOrPanic(dirName, fileName string) string {
+	name := filepath.Join(dirName, fileName)
+	file, err := os.OpenFile(name, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0600)
```

Comment:
why not simply `os.Create(...)`? The only difference would be that it is created using `0666` mode vs `0600`, but I don't think that matters here.